### PR TITLE
Allow the layout to be resized horizontally

### PR DESCRIPTION
**Pull request by wooferzfg**
This resolves some of the issues in #46. There's no saving at the moment since this is LSO only and nothing is in livesplit-core.

### CODE REVIEW

File: src/css/Layout.scss [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
   font-family: "fira", sans-serif;
   font-size: 15px;
-  width: 300px;
   text-shadow: 2px 2px 2px rgba(0, 0, 0, 0.5);
+  position: relative;
   -webkit-touch-callout: none;
   -webkit-user-select: none;
   -khtml-user-select: none;
   -moz-user-select: none;
   -ms-user-select: none;
```
Comment (CryZe) [inline src/css/Layout.scss]: Do we need these? I don't think we support any of these ancient browsers anymore (we should check if any is still needed though)

**Comment by CryZe** [comment]
I feel like we want to adjust the buttons to have the same width:

**Comment by CryZe** [comment]
Other than that, this seems REALLY good. I'm surprised how well this works.

**Comment by CryZe** [comment]
I fixed that. However something that I noticed is that the sidebar (independent of my changes) doesn't work well anymore. If it's over the timer, then the click region of the timer is preferred over the buttons on the sidebar.

**Comment by CryZe** [comment]
I fixed that too. So now I would merge it, but I'd like you to review my changes as well.

**Comment by wooferzfg** [comment]
The button margins were previously based on percentages rather than being consistent, so I fixed that.

File: src/css/Layout.scss (hunk 1 above)
Comment (wooferzfg) [reply src/css/Layout.scss]: Based on the [compatiblity table](https://developer.mozilla.org/en-US/docs/Web/CSS/user-select), most browsers support the `user-select` properly, but a lot of them require it to be `-webkit-user-select`.

chrome_fqQJpyqebA

Based on this, I think we can keep `-webkit-touch-callout` and `-webkit-user-select` but get rid of the rest.

**Comment by wooferzfg** [comment]
@CryZe This looks good to me now. Feel free to merge if you think it's ready.

**Pull request merged**